CONTRIBUTING.md

```diff
@@ -77,6 +77,21 @@ Ie. if you want to install `mocking` into `graphql`, run:
 lerna add @vergunningcheck/mocking --scope=graphql
 ```
 
+## Automated tests
+
+When you're making a new PR for a feature or a bug, it says "Please make sure you added the necessary automated tests." Please help us by:
+
+- adding unit tests (to atoms / components / utils)
+- adding or extending the e2e tests, when you're working on new routes or pages (see [packages/e2e/README.md](./packages/e2e/README.md))
+- fixing and extending failing tests
+- writing tests in (or converting tests to) TypeScript
+
+These commands may help you:
+
+- `npm run test:coverage` - to test the coverage for all files
+- `npm run test -- src/atoms --collect-coverage` - to test the coverage for certain files (in packages/client)
```

Comment:
Some of these tips are specific to `packages/client`, can you somehow make that more clear?